### ISSUE TITLE
Bar Chart Date Issue

### ISSUE THREAD

**Issue by dcagnetta** [SparkDevNetwork/Rock#1669]
Hi guys

i created a bar chart which has 3 metric values for a specific month

the dates however show incorrectly

![image](https://cloud.githubusercontent.com/assets/12990665/17358150/b549fc3a-5960-11e6-9607-67bb0bf813d9.png)

**Comment by arranf** [comment]
Can you post some specific steps to recreate this? What version of Rock are
you on?

**Comment by dcagnetta** [comment]
Hi Arran

I simple posted 3 metric values for a specific campus with 3 different dates

I am testing 5.2

**Comment by arranf** [comment]
@dcagnetta Can you share the date format with us?

**Comment by mikejed** [comment]
On 6.x and on v8 prealpha, I have provided a few metric values. (I used Adult Attendance at 9am on the Main Campus). First, I put three values within the same month and asked a bar graph block (which I'd configured for that specific set of partitions) to show me data from the current year.

I got a single bar on the graph, and NO dates (the x axis is labeled `Main Campus, Sunday at 9:00 am`). Hovering over the graph displays a tooltop indicating the earliest dated metric I entered, with a value of 300, even though the bar itself graphically reaches up to nearly 3,000 (the total of all the values I put into the metric).

I changed one of the values to a different month, and observed no change in the graph.

I created a new (non-partitioned) metric, and the x axis now reads "(null)". I created an arbitrary partition instead, and assigned values to the metric values. Now I get different bars, one for each partition value. But I still get only one bar and the tooltip for only one value when hovering over a bar that actually represents the sum of several values.

I'm unable to reproduce the graph which @dcagnetta originally posted - that looks like three bars stacked on top of each other, which I'm not able to reproduce. I also do not see a partition type which would create a date on the x axis. I do believe that the tooltips are incompletely-implemented since they give a single value that does not match the "height" of the bar.

However, all of this is somewhat consistent with the warning that the "dashboard" lava blocks are not ready for primetime, and are subject to change. The clarification I would pose of the OP was whether the data was partitioned, as it appears this bar graph block is designed to break data up by partition, NOT by time period.

![image](https://user-images.githubusercontent.com/6932047/38007759-cd9cdfb8-31ff-11e8-942e-52ccf2e5bd72.png)

If we cannot reproduce the date issue the OP referred to in the x axis of the chart, this issue should probably be closed. However, another issue may need to be opened in it's place regarding the bar chart block itself, as there do appear to be some issues with it.
